<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interactWindow</class>
 <widget class="QWidget" name="interactWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>820</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>← Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageNameLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: orange;
font-size: 35px;
font-family: Roboto;</string>
   </property>
   <property name="text">
    <string>Train circle lines</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainLineNameLabel">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>150</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #595959;
	font-size: 30px;
	font-family: Roboto;
	background-color: #d9d9d9;
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addLineButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>➕  line</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNameInput">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addStButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>➕  station</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> k = </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="removeStKInput">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>540</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="removeStButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>❌  station</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>710</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> k = </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="changeStKInput">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>710</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="changeStButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>630</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>⚙  change station</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>680</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="changeStNameInput">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>500</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeOutLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>495</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #d9d9d9;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loadKUTButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>630</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Kyiv Urban Train</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>640</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>load:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadMCCButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>690</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Moscow Central Circle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadMMCButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>750</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Moscow Metro Circle</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="trainLineTable">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>210</y>
     <width>450</width>
     <height>660</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size: 25px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
	padding-left: 30px;
}</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fromSelectionRadioButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>from</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="toSelectionRadioButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="toStationLabel">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
	font-size: 17px;
	font-family: Roboto;
	background-color: #e1e1e1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fromStationLabel">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
	font-size: 17px;
	background-color: #e1e1e1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="addStTimeNextInput">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>432</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time to next = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addStNameInput">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>k = </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="removeStKInput_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>458</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: red;
	font-size: 17px;
	font-family: Roboto;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
